coursier.bzl

```diff
@@ -184,10 +184,46 @@ def generate_imports(repository_ctx, dep_tree, srcs_dep_tree = None):
 
             all_imports.append("\n".join(target_import_string))
 
-            # Also create a versionless alias target
+            # 6. Create a versionless alias target
+            #
+            # alias(
+            #   name = "org_hamcrest_hamcrest_library",
+            #   actual = "org_hamcrest_hamcrest_library_1_3",
+            # )
             target_alias_label = _escape(_strip_packaging_and_classifier_and_version(artifact["coord"]))
```

Comment:
while you are here.. let's rename target_alias_label to versionless_target_alias_label to make it more obvious that we're working with a versionless alias.